building/configlet/lint.md

```diff
@@ -181,7 +181,7 @@ The `config.json` file should have the following checks:
 - The `"key_features"` key is optional
 - The `"key_features"` value must be an array with length = 6
 - The `"key_features[].icon"` key is required
-- The `"key_features[].icon"` value must be a string that matches one of the pre-defined icon values (TODO: add link to list of icons)
+- The `"key_features[].icon"` value must use one of the [pre-defined icon values](../tracks/config-json.md#keyfeatures)
```

Comment:
```suggestion
- The `"key_features[].icon"` value must use one of the [pre-defined icon values](../tracks/config-json#keyfeatures)
```

IIRC this should this omit the `.md` extension. (There are others in this file, but I'll eventually fix them in https://github.com/exercism/docs/pull/160).